app/schemas/users.js

```diff
@@ -66,3 +66,22 @@ exports.userLoginSchema = {
     errorMessage: ErrorMessages.PASSWORD_NOT_COMPLY
   }
 };
+
+exports.getUser = {
+  page: {
+    in: ['query'],
+    isInt: true,
+    isEmpty: {
```

Comment:
These parameters should be optional, so you do not always have to send the `page` and the `limit`.